Doc/library/sys.rst

```diff
@@ -1787,29 +1827,28 @@ always available.
    A :term:`named tuple` holding information about the thread
    implementation.
 
-   .. tabularcolumns:: |l|p{0.7\linewidth}|
-
-   +------------------+---------------------------------------------------------+
-   | Attribute        | Explanation                                             |
-   +==================+=========================================================+
-   | :const:`name`    | Name of the thread implementation:                      |
-   |                  |                                                         |
-   |                  |  * ``'nt'``: Windows threads                            |
-   |                  |  * ``'pthread'``: POSIX threads                         |
-   |                  |  * ``'pthread-stubs'``: stub POSIX threads              |
-   |                  |    (on WebAssembly platforms without threading support) |
-   |                  |  * ``'solaris'``: Solaris threads                       |
-   +------------------+---------------------------------------------------------+
-   | :const:`lock`    | Name of the lock implementation:                        |
-   |                  |                                                         |
-   |                  |  * ``'semaphore'``: a lock uses a semaphore             |
-   |                  |  * ``'mutex+cond'``: a lock uses a mutex                |
-   |                  |    and a condition variable                             |
-   |                  |  * ``None`` if this information is unknown              |
-   +------------------+---------------------------------------------------------+
-   | :const:`version` | Name and version of the thread library. It is a string, |
-   |                  | or ``None`` if this information is unknown.             |
-   +------------------+---------------------------------------------------------+
+   .. attribute:: thread_info.name
+
+      The name of the thread implementation:
+
+      * ``"nt"``: Windows threads
```

Comment:
This bulleted list might look nicer as a table (same for the one under thread_info.lock)

Reply:
I prefer the bullet version:

<details>
<summary>Bullets</summary>

<img width="814" alt="Screenshot 2023-08-20 at 08 37 51" src="https://github.com/python/cpython/assets/13780613/17d68044-9b9b-45f8-a3a4-1dd8116fe8e5">

</details>

<details>
<summary>Tables</summary>

<img width="814" alt="Screenshot 2023-08-20 at 08 40 00" src="https://github.com/python/cpython/assets/13780613/d876623b-f861-457e-a387-18c4b27601a7">

</details>

Reply:
The table version, to me, describes the data more naturally: the data is essentially a one-to-one mapping between the value you could have for `thread_info.name` and the meaning of that particular value. Tables are commonly used for describing mappings in the CPython docs and other docs.

_But_ I don't feel too strongly about it, and the status quo wins in cases of disagreement, so we can leave this if you disagree :)

Reply:
Another alternative would be to use a definition list? (I also prefer bullets, sorry Alex!)

A

Reply:
> (I also prefer bullets, sorry Alex!)

No worries, I'm obviously outvoted on this one 😄

Reply:
Comparison with a definition list:

* <details>
  <summary>Bullets</summary>

  ![image](https://github.com/python/cpython/assets/9087854/d04b4ac9-09be-4dbd-afa8-01c93f942f37)

  </details>
* <details>
  <summary>Tables</summary>

  ![image](https://github.com/python/cpython/assets/9087854/65488a11-4a45-450b-a115-4eee626a693c)

  </details>
* <details>
  <summary>Definition Lists</summary>

  ![image](https://github.com/python/cpython/assets/9087854/5ae451a5-5b34-4dce-a811-ae2ec5121801)

  </details>

A

Reply:
I still like the tables best here, but the definition list is also nice in my opinion!

Reply:
I have no strong opinion here :) I agree that tables are nice, but I don't like they way they align.